rn/Teacher/src/routing/Navigator.js

```diff
@@ -68,7 +68,7 @@ export default class Navigator {
     }
     if (options.modal) {
       const embedInNavigationController = options.embedInNavigationController == null || options.embedInNavigationController
-      const disableSwipeDownToDismissModal = options.disableSwipeDownToDismissModal || false
+      const disableSwipeDownToDismissModal = options.disableSwipeDownToDismissModal || true
```

Comment:
This is a little confusing because passing `{ disableSwipeDownToDismissModal: false }` will be ignored. I think we should either remove the option entirely or make it possible to override the default behavior.